src/handlers/comment/handlers/multiplier.ts

```diff
@@ -49,7 +49,10 @@ export const multiplier = async (body: string) => {
         reason += part.replace(/['"]/g, "") + " ";
       }
     }
-
+    if (username === "") {
```

Comment:
I question this implementation. I personally would probably deliberately parse "no user" as `username === null` to express that the user value has been recognized by the system, and that it has been assigned to nothing.

Reply:
Thanks, but make sure that the code actually works with this change. I would have expected that you change it to `null` somewhere else in the code when the value is being set.